scipy/conftest.py

```diff
@@ -129,6 +129,14 @@ def check_fpu_mode(request):
     except ImportError:
         pass
 
+    try:
+        #XXX: change import to `jax` before merge
```

Comment:
marking this - the main JAX namespace is missing some functions like `asarray` which makes lots of our test code error out. Testing with `jax.experimental.array_api` for now but we'll need to switch at some point.

Reply:
The `jax` namespace will never support the array API, but eventually `jax.numpy` will (it currently supports most things, including `asarray`).

Reply:
right. That should be enough for now to test that `jax.numpy` arrays are correctly handled by `array_api_compat` and the `jax.experimental.array_api` namespace is returned. Thanks!